src/robotide/utils/eventhandler.py

```diff
@@ -41,6 +41,13 @@ def start_listening(self, path):
         if self._initial_watched_path != path:
             self._initial_watched_path = path
         self.stop_listening()
+        # on MSW we get a popup from wxWidgets
+        # (https://github.com/wxWidgets/wxWidgets/blob/master/src/msw/fswatcher.cpp#L165)
+        # when the path is a network share, like for example WSL: \\wsl.localhost\docker-desktop\tmp\
+        # We avoid the popup by ignoring it
+        if path.startswith('\\\\') and os.sep == '\\':
+            print(f"INFO: Not watching file system changes for path: {path}")
```

Comment:
I am not seeing this message in the RIDE log. Is it supposed to show up there?

Reply:
It shows in the console/terminal/shell if you start from there :).